autoload/gtfo/open.vim

```diff
@@ -66,7 +66,8 @@ endf
 func! s:force_cmdexe() abort
   if &shell !~? "cmd" || &shellslash
     let s:shell=&shell | let s:shslash=&shellslash | let s:shcmdflag=&shellcmdflag
-    set shell=$COMSPEC noshellslash shellcmdflag=/c
+    exec 'set shell='.$COMSPEC
```

Comment:
could you try this instead:

    let &shell=$COMSPEC

Reply:
that works too. in both vim 8 and neovim on windows.